lib/python/ensembl/brc4/runnable/download_assembly_data.py

```diff
@@ -35,11 +35,11 @@ def param_defaults() -> Dict[str, Any]:
 
     def run(self):
         accession = self.param_required('accession')
-        main_download_dir = self.param_required('download_dir')
-        download_dir = main_download_dir + "/" + accession
+        main_download_dir = Path(self.param_required('download_dir'))
+        download_dir = main_download_dir / accession
 
         # Set and create dedicated dir for download
-        if not os.path.isdir(download_dir):
+        if not download_dir.is_dir():
```

Comment:
Same as before, the next line can be changed to `download_dir.mkdir(parents=True)`. If you search for the remaining usages of `os.`, you will find a few additional changes that can be done to remove `import os`.